test/lib/ach_client/objects/return_codes_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
+
+class ReturnCodesTest < MiniTest::Test
+  def test_unauthorized
+    unauthorized_codes = AchClient::ReturnCodes.unauthorized
+    assert(unauthorized_codes.all?{|code| code.unauthorized_return?})
```

Comment:
nitpick feel free to ignore
```suggestion
    assert(unauthorized_codes.all?(&:unauthorized_return?)
```